main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -90,9 +90,9 @@ object Bootstrap {
       s"""*.fpkg
          |*.jar
          |.GITHUB_TOKEN
-         |artifact/
-         |build/
-         |lib/
+         |$artifactDirectoryRaw
```

Comment:
Why doesn't this use `getArtifactDirectory`?

Reply:
it just makes the code worse to read. How do i get `artifacts/` out from a path, exactly that format? `getArtifactDirectory` uses the raw string, so there is only one source of truth

Reply:
also note that `artifactDirectoryRaw` is private

Reply:
Use `getFileName`?